docs/resources/apig_application_authorization.md

```diff
@@ -0,0 +1,61 @@
+---
+subcategory: "API Gateway (Dedicated APIG)"
+---
+
+# huaweicloud_apig_application_authorization
+
+Using this resoruce to authorize APIs for application, allowing it to access the published APIs within HuaweiCloud.
+
+-> For an application, an environment can only create one `huaweicloud_apig_application_authorization` resource.
```

Comment:
What the `environment` means here?

Reply:
all published APIs must belong to an environment.